edx/edx-enterprise
# [ENT-300] Data Sharing Consent: Expand Arrow off by a couple pixels

**Description:** This PR fixes the CSS for the expand arrow in the data sharing consent page.

**JIRA:** [ENT-300](https://openedx.atlassian.net/browse/ENT-300)

**Dependencies:** None

**Installation instructions:** Basic enterprise installation

**Testing instructions:**

1. Set up an EnterpriseCustomer, enable data sharing consent, and set the consent requirement level to "at login".
2. Add a user to the EnterpriseCustomer using the Manage Learners admin panel.
3. Log in as the learner and you'll be redirected to the data sharing consent page.
4. Click the arrow in the Data Sharing Policy bar.
5. Verify that once the policy is displayed the arrow doesn't look like this:

![before](https://cloud.githubusercontent.com/assets/560781/24980073/d1f83334-1f93-11e7-94df-56add7dada38.png)

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**

**Reviewers:**
